packages/react-effect/package.json

```diff
@@ -23,10 +23,11 @@
   },
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-effect/README.md",
   "dependencies": {
-    "react-tree-walker": "^4.3.0",
+    "@shopify/useful-types": "^1.1.0",
```

Comment:
`react-dom` needs to be here, right? Or is it a peer dependency?

Reply:
Good catch, need to add it to peer dependencies